src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -291,7 +321,7 @@ bool ansi_c_entry_point(
   const code_typet::parameterst &parameters=
     to_code_type(symbol.type).parameters();
 
-  if(symbol.name==standard_main)
+  if(symbol.name==ID_main)
```

Comment:
Then I'd suggest to also drop standard_main completely from the codebase. Could you please add a commit doing so?